sBTC SIP

```diff
@@ -0,0 +1,166 @@
+    - Returns the sBTC to the holder.
+    - Records the signer votes.
+
+To support deposits and withdrawals in sBTC, Signers must collectively make smart contract calls on Stacks. We require each signer to individually sign any contract call and ensure at least 70% of the signer signatures are present for each contract call to be valid.
+
+## Auxiliary Features
+Auxiliary features of the sBTC Bootstrap protocol are described below.
+
+- **Stacks Transaction Fee Sponsorship:** sBTC will include the option to have sBTC transactions on Stacks be sponsored in return for some sBTC. Using the approach suggested in this issue, sBTC users will be able to nearly spend sBTC as gas by getting support from an existing STX holder.
+- **Signer Wallet Rotation:** Mechanisms are provided for the scenario where a signer wants to rotate their key. For this to happen, signers must coordinate offline and vote on-chain on the new signer set (aka set of keys). Once the new signer set is determined, the signers conduct a wallet handoff and re-execute DKG.
+
+### sBTC Bootstrap Signer Eligibility
+The Stacks 2.5 epoch activated the Stacks Signer network, whereby Stackers (users who lock STX in consensus in order to earn BTC rewards) are able to participate in Stacks consensus. Stackers vote to append blocks to the Stacks blockchain by producing a signature over the block. Each Stacker's share of the signature (its weight) is proportional to the fraction of locked STX it owns. This essential function ensures the security and liveness of the Stacks network.
+
+The sBTC Bootstrap Signers will be a separate subset of Stacks Signers. For the sBTC Bootstrap release, Signers will run a binary in addition to the core Stacks software. There will be 15 signers participating in the sBTC bootstrap release. In a subsequent release and hard fork, sBTC will be enshrined in Stacks consensus and Signers will only need to maintain a single binary.
+
+The following eligibility criteria has been used to identify the sBTC Bootstrap Signers:
+- **Stacks 2.5 Participation:** Active participation running a signer on Stacks 2.5 testnet or mainnet for at least two previous cycles.
+- **Technical Performance and Uptime:** Consistency in operational status and network stability. Signers should demonstrate a target uptime of at least 99%.
+- **Communication & Availability:** Running an sBTC Signer is a pivotal role in the Stacks ecosystem, which requires a high degree of availability and communication with Stacks core developers. Signers must be able to respond to updates within 24 hours.
+- **Ecosystem Alignment:** Commitment to the growth and principles of the Stacks and Bitcoin ecosystems with contributions to advance the network.
+
+By implementing a whitelisted signer set for the sBTC Bootstrap release, this helps to reduce time to market to launch sBTC sooner, remove complexity from the initial protocol design, and ensure the sBTC protocol is secure at launch.
```

Comment:
Addressing "time to market" is not (yet) in the purview of the SIP process.  There's no CAB for it, for example.